ouroboros-network/src/Ouroboros/Network/NodeToNode.hs

```diff
@@ -287,24 +317,29 @@ ipSubscriptionWorker
           peerid
           NodeToNodeProtocols
           IO BL.ByteString x y)
-    -> IO void
+    -> IO ()
```

Comment:
What's the reason for this change? (throughout this file)

Reply:
#902 Was written with ().

Reply:
ok. @coot ^

Reply:
I'd prefer a `void` or `Void`

Reply:
Addressed